stylus/cozy-ui/build.styl

```diff
@@ -924,6 +924,23 @@ Display an chip that represents complex identity
  Styleguide utilities.text.ellipsis
 */
 
+/*
+ Space Ellipsis
+
+ Adds ellipsis to a long text but keep breaking spaces<br>
```

Comment:
I think the description  might be off 😅. What's a breaking space? Did you mean line breaks instead?

Reply:
Les "breaking spaces" sont des espaces entre les mots qui peuvent provoquer un retour à la ligne, contrairement au "no wrap" qui a un comportement "non breaking spaces" c'est à dire que les espaces entre les mots ne provoquent pas de retour à la ligne. On se retrouve donc sur une seule ligne, ce qui n'est pas le cas ici, les "espaces sécables" sont conservés. C'est un terme typographique, mais peut-être pas clair ?

Reply:
Ok au temps pour moi, c'est le`<br>` qui m'a induit en erreur mais ca fait pas partie de la doc 🤦‍♂️. Juste une petite typo à corriger du coup.